src/ui/common/src/components/workflows/version_selector.tsx

```diff
@@ -83,16 +108,42 @@ export const VersionSelector: React.FC = () => {
 
   return (
     <Box sx={{ display: 'flex', alignItems: 'center' }}>
-      <FormControl sx={{ minWidth: 120 }} size="small">
-        <Select
-          sx={{ maxHeight: 48 }}
-          id="grouped-select"
-          autoWidth
-          value={selectedResultIdx}
-        >
-          {getMenuItems()}
-        </Select>
-      </FormControl>
+      <Button
+        sx={{
+          backgroundColor: !!menuAnchor ? theme.palette.gray[50] : 'white',
+          ':hover': {
+            backgroundColor: theme.palette.gray[50],
+          },
+          p: 1,
```

Comment:
why we have both `p` and `pr` here?

Reply:
Good catch. This was a remnant of something I was trying out.